db_stress_tool/db_stress_test_base.cc

```diff
@@ -1704,10 +1698,13 @@ Status StressTest::TestIterateImpl(ThreadState* thread,
       op_logs += "S " + key.ToString(true) + " ";
     }
 
-    if (IsRetryableInjectedError(iter->status()) ||
-        IsRetryableInjectedError(cmp_iter->status())) {
-      return !iter->status().ok() ? iter->status() : cmp_iter->status();
+    if (!iter->status().ok() && IsRetryableInjectedError(iter->status())) {
```

Comment:
From the name, I would assume `IsRetryableInjectedError(s)` implies `!s.ok()`. Should we make that assumption / simplification?

Reply:
yes

Reply:
We fix the name so it's obvious that the parameter s is an error

Reply:
Fixed